### PR TITLE
Create `ru_ru.json`

### PR DESCRIPTION
**Pull request by mpustovoi**
Created `ru_ru.json`

### CODE REVIEW

**Comment by xllifi** [comment]
Good work, I like it

File: src/main/resources/assets/expandeddelight/lang/ru_ru.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	"block.expandeddelight.chili_pepper_crop": "Перец чили",
+	"block.expandeddelight.wild_peanuts": "Дикий арахис",
+	"block.expandeddelight.peanut_crop": "Арахис",
+	"block.expandeddelight.cinnamon_sapling": "Саженец корицы",
+	"block.expandeddelight.cinnamon_log": "Коричное бревно",
+	"block.expandeddelight.salt_ore": "Соляная руда",
+	"block.expandeddelight.deepslate_salt_ore": "Глубинносланцевая соляная руда",
+	"block.expandeddelight.juicer": "Соковыжималка",
+	"block.expandeddelight.mortar_and_pestle": "Ступка и пестик",
+	"block.expandeddelight.mortar_and_pestle.pass": "Это не то, что ты можешь сокрушить…",
```
Comment (xllifi) [inline src/main/resources/assets/expandeddelight/lang/ru_ru.json]: This line (63) though. I don't like sentence structure and "сокрушить" is really a wrong word.
I'd replace it with shorter "Это нельзя измельчить..."

**Review by xllifi — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/expandeddelight/lang/ru_ru.json (hunk 1 above)
Comment (mpustovoi) [reply src/main/resources/assets/expandeddelight/lang/ru_ru.json:63]: I agree.

Comment (mpustovoi) [reply src/main/resources/assets/expandeddelight/lang/ru_ru.json]: > This line (63) though. I don't like sentence structure and "сокрушить" is really a wrong word. I'd replace it with shorter "Это нельзя измельчить..."

I agree.

**Pull request merged**